app/angular/src/client/preview/types.ts

```diff
@@ -29,7 +29,6 @@ export interface StoryFnAngularReturnType {
   /** @deprecated `component` story input is deprecated, and will be removed in Storybook 7.0. */
   component?: any;
   props?: ICollection;
-  propsMeta?: ICollection;
```

Comment:
`StoryFnAngularReturnType` is part of the public API (as it is used in `Story` and `Meta` in `types-6-0.ts`) so even if `propsMeta` isn't used anymore inside Storybook I think we should deprecate it and remove it in 7.0 to limit the number of breaking changes in 6.2. 

BUT if I remember correctly we named this file `types-6-0` to keep the option to make breaking changes in the types so 🤷🏻 .

What's your opinion on that @shilman @ndelangen?

Reply:
indeed, it may be wiser to add a simple @deprecated 

When is planed the version 7 ?

Reply:
Probably during the summer. I agree deprecating is easier.